polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -861,10 +893,6 @@ mod benches {
 		[pallet_child_bounties, ChildBounties]
 		[pallet_claims, Claims]
 		[pallet_collective, Council]
-		// Fixme #177: For some reason the benchmarks write both to the same pallet_collective weight
-		// file. No reason to run both before we fix that, as the pallet_collective benchmark
-		// takes the longest.
-		// [pallet_collective, TechnicalCommittee]
 		[pallet_democracy, Democracy]
```

Comment:
what is the resolution here?

Reply:
I see no problem which needs solving. both instances can take their weights from the same benchmark. its the same pallet so it's the same weights...